osbrain/tests/test_proxy.py

```diff
@@ -119,6 +122,41 @@ def ping(self):
         InitTimeoutProxy('foo', timeout=1.)
 
 
+def test_agent_proxy_wait_running(nsproxy):
+    """
+    Using wait_for_running on a proxy after initialization should block until
```

Comment:
You could use backticks ( \` ) around \`wait_for_running\` in the docstring. You could also add a dot and/or parenthesis as in `.wait_for_running()`, but that only in case you prefer that style.

Reply:
They have in reST/Sphinx, which is the format we are using for docs generation. Things surrounded by backquotes can be interpreted as code/names and even automatically linked.